src/PIL/Image.py

```diff
@@ -1447,6 +1454,49 @@ def _reload_exif(self):
         self._exif._loaded = False
         self.getexif()
 
+    def get_child_images(self):
+        child_images = []
+        exif = self.getexif()
+        ifds = []
+        if ExifTags.Base.SubIFDs in exif:
+            subifd_offsets = exif[ExifTags.Base.SubIFDs]
+            if subifd_offsets:
+                if not isinstance(subifd_offsets, tuple):
+                    subifd_offsets = (subifd_offsets,)
+                for subifd_offset in subifd_offsets:
+                    ifds.append((exif._get_ifd_dict(subifd_offset), subifd_offset))
+        ifd1 = exif.get_ifd(ExifTags.IFD.IFD1)
+        if ifd1 and ifd1.get(513):
+            ifds.append((ifd1, exif._info.next))
+
+        offset = None
+        for ifd, ifd_offset in ifds:
+            current_offset = self.fp.tell()
+            if offset is None:
+                offset = current_offset
+
+            fp = self.fp
+            thumbnailOffset = ifd.get(513)
+            if thumbnailOffset is not None:
+                try:
+                    thumbnailOffset += self._exif_offset
+                except AttributeError:
+                    pass
+                self.fp.seek(thumbnailOffset)
+                data = self.fp.read(ifd.get(514))
+                fp = io.BytesIO(data)
+
+            with open(fp) as im:
+                if thumbnailOffset is None:
```

Comment:
snake_case

```suggestion
            thumbnail_offset = ifd.get(513)
            if thumbnail_offset is not None:
                try:
                    thumbnail_offset += self._exif_offset
                except AttributeError:
                    pass
                self.fp.seek(thumbnailOffset)
                data = self.fp.read(ifd.get(514))
                fp = io.BytesIO(data)

            with open(fp) as im:
                if thumbnail_offset is None:
```

Reply:
`self.fp.seek(thumbnailOffset)` was missed in the middle, so I've applied this as a separate commit